src/main/java/ColourSupplier.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Random;
+
+public class ColourSupplier {
+    private String[] colours = new String[]{
+            "brown", "blue", "yellow", "white", "black", "green", "grey", "orange"
+    };
+
+    public String[] getColours() {
+        return colours;
+    }
+
+    public void setColours(String[] colours) {
+        this.colours = colours;
+    }
+
+    public static String getColour(String[] colours) {
+        Random random = new Random();
+        int randomColour = random.nextInt(colours.length);
+        return colours[randomColour];
+    }
```

Comment:
```suggestion
    public String getColour() {
        Random random = new Random();
        int randomColour = random.nextInt(colours.length);
        return colours[randomColour];
    }
```
please don’t create static methods - they are in general bad practice